shell/common/node_bindings.cc

```diff
@@ -122,6 +123,67 @@ void stop_and_close_uv_loop(uv_loop_t* loop) {
 
 bool g_is_initialized = false;
 
+bool IsPackagedApp() {
+  base::FilePath exe_path;
+  base::PathService::Get(base::FILE_EXE, &exe_path);
+
+#if defined(OS_WIN)
+  std::string path = base::UTF16ToUTF8(exe_path.value());
+#else
+  std::string path = exe_path.value();
+#endif
+
+  std::transform(path.begin(), path.end(), path.begin(), ::tolower);
+
+#if defined(OS_WIN)
+  return path == "electron.exe";
```

Comment:
I believe this should be `!=`.

Reply:
we want this to return `true` when it _is_ packaged though, so i think this is what we want to check?

Reply:
I think when app is packaged the executable name should not be `electron`? The `App.isPackaged` is implemented so.